rook/Tester.py

```diff
@@ -367,14 +366,13 @@ def get_valid_params():
     params.add_param('output_wait_time', '-1', 'Number of seconds to wait for output')
     return params
 
-  def __init__(self, name, params):
+  def __init__(self, _name, params):
     """
       Initializer for the class.  Takes a String name and a dictionary params
       @ In, name, string, name of the class
```

Comment:
name --> _name

Reply:
And the reason I switched name to _name is because then python (and pylint) know that it is okay that it is an unused parameter.

Reply:
Docstring updated.